nginx/nginx.conf

```
@@ -89,8 +89,12 @@ http {
 			proxy_pass http://relay;
 		}
 		location ^~ /js-sdk/ {
-			autoindex on;
-			root  /var/www/js-sdk;
+			include mime.types;
+			autoindex off;
+			etag on;
+			root  /var/www/;
+			try_files $uri =404;
+			add_header Access-Control-Allow-Origin *;
```

Yeah, probably because sentry.io cannot add all domains and subdomains users are using. Although I'm not sure, and if it responds to any random subdomain which users point to sentry.io.

Anyway I'd suggest at least adding a comment stating that it's recommended for users to change this.
```suggestion
			add_header Access-Control-Allow-Origin *; # it is recommended to change this to your sentry  (in most cases same as system.url-prefix)
```